### PR TITLE
fix: use local node-version

### PR DESCRIPTION
**Pull request by amazingandyyy**
**Description**

- remove the not-good node-version-data
- use nodeVersions directly

**Changes**

* fix: use local node-version

🚀 PR created with [fotingo](https://github.com/tagoro9/fotingo)

### CODE REVIEW

File: node.js [hunk 1]
```diff
@@ -87,13 +84,8 @@ export default class Node extends Tool {
...
      */
-    async getVersionData() {
-        try {
-            return await getVersionData()
-        } catch (e) {
-            this.warning(`Failed to download Node version data: ${e.message}`)
-            return nodeVersions
-        }
+    getVersionData() {
+        return nodeVersions
```
Comment (tagoro9) [inline node.js:90]: Add a huge TODO here to add back the logic to fetch node versions from Nodejs

**Review by tagoro9 — COMMENTED** [review]
You need to run `npm run prepare` and commit the changes

File: node.js (hunk 1 above)
Comment (amazingandyyy) [reply node.js:90]: updated

**Review by tagoro9 — APPROVED** [review]

File: tool.test.js [hunk 2]
```diff
@@ -83,13 +83,14 @@ describe("version", () => {
         expect(version).toMatch(/^\d+\.\d+\.\d+$/)
     })
 
-    // Skipping this test because it makes tests look like they fail
-    it("errors sensibly if a thing isn't found", () => {
```
Comment (tagoro9) [inline tool.test.js:87]: Does this test need to be commented out?

**Pull request merged**